chain/consensus/mir/types.go

```diff
@@ -130,23 +127,38 @@ func MessageBytes(msg MirMessage) ([]byte, error) {
 	return append(msgBytes, byte(msgType)), nil
 }
 
-// Mir's checkpoint period is computed as the number of validators times the SegmentLength.
-// In order to configure the initial checkpoint period close to a specific value, we need
-// to set the SegmentLength for the SMR system accordingly. This function does this math
-// for you.
-func segmentForCheckpointPeriod(desiredPeriod int, membership map[t.NodeID]t.NodeAddress) (int, error) {
-	segment := desiredPeriod / len(membership)
-	if segment < 1 {
-		return 0, fmt.Errorf("wrong checkpoint period: the minimum checkpoint allowed for this number of validators is %d", len(membership))
-	}
-	return segment, nil
-}
-
 type ParentMeta struct {
```

Comment:
CAn we add a description of what is this type here?

Reply:
That is not mine )